src/mpcCoreKit.ts

```diff
@@ -467,7 +467,10 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
       await this.backupMetadataShare(factorKey);
       await this.addFactorDescription(factorKey, shareDescription, additionalMetadata);
       if (!this.options.manualSync) await this.tKey.syncLocalMetadataTransitions();
-      return factorKey.toArrayLike(Buffer, "be", SCALAR_LEN).toString("hex");
+      return factorKey
+        .toArrayLike(Buffer, "be", SCALAR_LEN)
+        .toString("hex")
+        .padStart(SCALAR_LEN * 2, "0");
```

Comment:
This padding is redundant. `toArrayLike(Buffer, "be", SCALAR_LEN)` already guarantees that it has the correct length.

Reply:
@matthiasgeihs , you are right